src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -300,6 +300,7 @@ public function beforeRedirect(Event $event, $url, Response $response)
                 'REQUEST_METHOD' => 'GET'
             ]
         ]));
+        $response->statusCode(200);
         $response->send();
         $response->stop();
     }
```

Comment:
Wouldn't that be a different PR, as this PR and its change has to be done either way to reset the status code back to 200?